src/irr/asset/CImageLoaderTGA.cpp

```diff
@@ -115,20 +115,21 @@ static void convertColorFlip(const core::smart_refctd_ptr<ICPUImage> &image, con
 {
 	const T *in = (const T *) src;
 	T *out = (T *) image->getBuffer()->getPointer();
-	
-	auto size = image->getCreationParameters().extent;
+	const auto extent = image->getCreationParameters().extent;
+	irr::core::vector3d size = {extent.width, extent.height, extent.depth};
```

Comment:
get the values with padding maybe? (or can you guarantee that you always use one region for the whole and only mip and its without padding)